### ISSUE TITLE
server isn't freezing 1.20.1

### ISSUE THREAD

**Issue by christoros** [QPCrummer/Slumber#7]
The server isn't actually freezing when it is supposed to. I am using version slumber 1.1.1 and carpet 1.4.112. 
Here is my console output:
```
slumber
[16:16:24] [Server thread/INFO]: Enabled: true, Frozen: true, Deeply: true
time query daytime
[16:16:35] [Server thread/INFO]: The time is 20631
time query daytime
[16:16:36] [Server thread/INFO]: The time is 20667
```

**Comment by QPCrummer** [comment]
Thank you for reporting this. I'll see what I messed up this time. Assuming it is a simple fix, it'll be fixed in a few hours

**Comment by QPCrummer** [comment]
Ok, so it actually appears to be an issue with Carpet's freeze functions. This may mean I need to either write my own custom freeze code _or_ I have to fix Carpet's

**Comment by QPCrummer** [comment]
@chrisdeman I have fixed the issue in version 1.1.2 https://modrinth.com/mod/slumber/version/1.1.2
The issue was with how I created my ServerTickRateManager from Carpet. It cannot be created as a new class, but rather requires and interface to operate

**Comment by christoros** [comment]
Great, thank you for your quick efforts!
